t/0-archive.t

```diff
@@ -0,0 +1,34 @@
+# Unit test for TEST_NGINX_ARCHIVE_PATH
```

Comment:
One way is to skip this test file entirely when there is no nginx visible in the PATH environment.